lib/OpenQA/Constants.pm

```diff
@@ -66,10 +66,11 @@ use constant {
     WORKER_SR_BROKEN        => 'worker broken',    # worker setup is generally broken, e.g. cache service not started
     WORKER_SR_DONE          => 'done',             # backend exited normally
     WORKER_SR_DIED          => 'died',             # backend died
+    WORKER_SR_FINISH_OFF    => 'finish-off',       # the worker is supposed to terminate after finishing assigned jobs
```

Comment:
I've chosen it because the worker will also turn itself off after finishing the current jobs. So I was going for the 2nd definition "to do the last part of something; to make something¹ end by doing one last thing²" from https://www.oxfordlearnersdictionaries.com/definition/english/finish-off?q=finish+off.

---

¹ execution of the worker
² processing assigned jobs